bin/cli.js

```diff
@@ -117,16 +123,19 @@ import i18nTransform from '../dist/transform.js'
       }
     }
 
+    let basePath = process.cwd()
```

Comment:
Could you make a PR with only that part of the code? I've missed that in my fix.